### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nithinmohantk**
Updated to Include Azure DevOps in places of Visual Studio Team Services

### CODE REVIEW

**Comment by geverghe** [comment]
@elbatk - Can you have a look at this?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-[![Visual Studio Team services](https://mseng.visualstudio.com/_apis/public/build/definitions/698eacea-9ea2-4eb8-80a4-d06170edf6bc/5908/badge)](https://mseng.visualstudio.com/vsts-cli/_build/index?context=mine&path=%5C&definitionId=6169&_a=completed)
+[![Azure DevOps Services](https://mseng.visualstudio.com/_apis/public/build/definitions/698eacea-9ea2-4eb8-80a4-d06170edf6bc/5908/badge)](https://mseng.visualstudio.com/vsts-cli/_build/index?context=mine&path=%5C&definitionId=6169&_a=completed)
 [![Python](https://img.shields.io/pypi/pyversions/vsts-cli.svg)](https://pypi.python.org/pypi/vsts-cli)
 
-# Microsoft Visual Studio Team Services CLI 
+# Microsoft Azure DevOps Services CLI 
 
-VSTS CLI is a new command line interface for [Visual Studio Team Services](https://www.visualstudio.com/team-services/) (VSTS) and [Team Foundation Server](https://www.visualstudio.com/tfs/) (TFS) 2017 Update 2 and later.
+Azure DevOps/VSTS CLI is a new command line interface for [Azure DevOps Services](https://www.visualstudio.com/team-services/) (VSTS) and [Azure DevOps Server](https://www.visualstudio.com/tfs/) (TFS) 2017 Update 2 and later.
```
Comment (wnjenkin) [inline README.md]: I don't think we should conflate these two names. Are you making these updates to clarify the change in name?

Comment (nithinmohantk) [reply README.md]: Yes. People might get confused, so referring to some places with both as Azure DevOps\VSTS CLI, I believe we could rename it to Azure DevOps CLI by omitting  VSTS CLI.

**Comment by geverghe** [comment]
We have created the [Azure DevOps Extension for Azure CLI](https://github.com/Microsoft/azure-devops-cli-extension). With this we have included the requisite documentation update to move all references to Azure DevOps. Closing this as it is no longer applicable. Please do let me know if you need any additional clarification.

**Pull request closed without merging**